lib/python/ensembl/brc4/runnable/download_assembly_data.py

```diff
@@ -14,9 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from pathlib import Path
```

Comment:
Cannot suggest changes outside your own changes, but this import section needs the same work as in `amend_genome_metadata.py`.